core/regexp/inspect_spec.rb

```diff
@@ -2,8 +2,10 @@
 
 describe "Regexp#inspect" do
   it "returns a formatted string that would eval to the same regexp" do
-    /ab+c/ix.inspect.should == "/ab+c/ix"
-    /a(.)+s/n.inspect.should =~ %r|/a(.)+s/n?|  # Default 'n' may not appear
+    not_supported_on :opal do
+      /ab+c/ix.inspect.should == "/ab+c/ix"
+      /a(.)+s/n.inspect.should =~ %r|/a(.)+s/n?|  # Default 'n' may not appear
```

Comment:
Since `n` only makes the Regexp have binary encoding, maybe it could be supported as a simple no-op?